build/packaging/linux/bin/kend

```diff
@@ -256,7 +356,7 @@ case "$1" in
         restart
         ;;
     *)
-        echo "Usages: kend {start|stop|restart|status}"
+        echo "Usages: kend {start|start-docker|stop|restart|status}"
```

Comment:
How about adding a `stop-docker`?
```suggestion
        echo "Usages: kend {start|start-docker|stop|stop-docker|restart|status}"
```

Reply:
@winnie-byun We could stop docker when we need to stop process. or we could share stop command for all. do you think there are any use-case with this?

Reply:
I just thought it would be convenient to start and stop with a same command. If you don't think this is not necessary, I am okay with your code. :)